common/main/java/com/couchbase/lite/MutableArray.java

```diff
@@ -82,11 +86,26 @@ public MutableArray(List<Object> data) {
     public MutableArray setData(List<Object> data) {
         synchronized (lock) {
             internalArray.clear();
-            for (Object obj : data) { internalArray.append(Fleece.toCBLObject(obj)); }
+            for (Object obj: data) { internalArray.append(Fleece.toCBLObject(obj)); }
             return this;
         }
     }
 
+    /**
+     * Populate an array with content from a JSON string.
+     * Allowed value types are List, Date, Map, Number, null, String, Array, Blob, and Dictionary.
+     * The List and Map must contain only the above types. Setting the new dictionary content will
+     * replace the current data including the existing Array and Dictionary objects.
+     *
+     * @param json the dictionary object.
+     * @return this Document instance
+     */
+    @NonNull
+    @Override
+    public MutableArray setJSON(@NonNull String json) {
```

Comment:
Not ToJSON? Is this because you are following java naming convention?

Reply:
setJSON() is in the MutableArray.  toJSON() is inherited from (immutable) Array.

Is that how you did it?